treescript/src/treescript/l10n.py

```diff
@@ -220,7 +220,7 @@ def build_commit_message(name, locale_map, dontbuild=False, ignore_closed_tree=F
     if not ignore_closed_tree:
```

Comment:
Nit: we should also perhaps move some of these assignments after this if, to avoid un-necessary operations if we have to return fast. Lines 214, 216, 217 and 218 can come afterwards.

Reply:
Even though this isn't much of a time-saver, I believe it's better for sanity check.
Hijacked @jcristau's PR to address this. Mea culpa for this.